src/test/java/com/hyperwallet/clientsdk/util/HyperwalletApiClientTest.java

```diff
@@ -1273,234 +1260,199 @@ public void testPost_200Response_ResponseContentTypeHeaderWithLeadingCharset() {
-            multiPart.bodyPart(data);
-
-            Client mockClient = createAndInjectWebResourceClient(hyperwalletApiClientEnc);
-
-            WebResource webResource = mock(WebResource.class);
-            when(mockClient.resource(baseUrl + "/documentUpload")).thenReturn(webResource);
-            WebResource.Builder builder = mock(WebResource.Builder.class);
-            when(webResource.type(MediaType.MULTIPART_FORM_DATA_TYPE)).thenReturn(builder);
-
-            InBoundHeaders headers = new InBoundHeaders();
-            headers.put(HttpHeaders.CONTENT_TYPE, Collections.singletonList("application/jose+json"));
-
-            ClientResponse clientResponse = mock(ClientResponse.class);
-
-            when(builder.put(ClientResponse.class, multiPart)).thenReturn(clientResponse);
-            when(clientResponse.getStatus()).thenReturn(200);
-
-            when(clientResponse.getEntity(String.class)).thenReturn("{result: \"Success\"}");
-            when(clientResponse.getHeaders()).thenReturn(headers);
-
-            hyperwalletApiClientEnc.put(baseUrl + "/documentUpload", multiPart, HyperwalletUser.class);
+            HyperwalletVerificationDocument multiPartUploadData = new HyperwalletVerificationDocument();
+            multiPartUploadData.setCategory("IDENTIFICATION");
+            multiPartUploadData.setCountry("US");
+            multiPartUploadData.setType("DRIVERS_LICENSE");
+            Map<String,String> uploadFiles = new HashMap<String,String>();
+            uploadFiles.put("drivers_license_front", "/integration/test.png");
+            multiPartUploadData.setUploadFiles(uploadFiles);
+            List<HyperwalletVerificationDocument> hyperwalletDocumentList = new ArrayList<>();
+            hyperwalletDocumentList.add(multiPartUploadData);
+            HyperwalletMultipartUtils.convert(hyperwalletDocumentList);
+
+            hyperwalletApiClient.put("https://api.sandbox.hyperwallet.com/rest/v4/users/test-user-token", HyperwalletMultipartUtils.convert(hyperwalletDocumentList), HyperwalletUser.class);
         } catch (Exception exception) {
-            assertThat(exception.getMessage(), is("java.text.ParseException: Invalid serialized unsecured/JWS/JWE object: Missing part delimiters"));
+            assertThat(exception.getMessage(), isOneOf("Server returned non-OK status: 401", "java.nio.file.NoSuchFileException: /integration/test.png"));
         }
     }
 
     @Test
     public void testUploadDocumentBusinessStakeholder() throws Exception {
-        FormDataMultiPart multiPart = new FormDataMultiPart();
+
+        Hyperwallet client = new Hyperwallet("test-username", "test-password");
+
+        String userToken = "user-token";
+        String businessStakeholderToken = "business-token";
+        HyperwalletBusinessStakeholder hyperwalletBusinessStakeholder = new HyperwalletBusinessStakeholder();
+        List<HyperwalletVerificationDocument> hyperwalletVerificationDocumentList = new ArrayList<>();
```

Comment:
No specific business logic in this class. ApiClient has no idea about HyperwalletVerificationDocument. Only Multipart object.